neo/Ledger/TrimmedBlock.cs

```diff
@@ -52,7 +51,9 @@ public Header Header
             }
         }
 
-        public override int Size => base.Size + ConsensusData.Size + Hashes.GetVarSize();
+        public override int Size => base.Size
+            /*Hashes*/ + Hashes.GetVarSize()
+            /*ConsensusData*/ + Hashes.Length > 0 ? ConsensusData.Size : 0;
```

Comment:
Is faster to check if is null

```suggestion
            /*ConsensusData*/ + ConsensusData != null ? ConsensusData.Size : 0;
```